webserver/views/test/test_datasets.py

```diff
@@ -201,3 +202,15 @@ def test_recording_info(self):
         self.assert200(resp)
         resp = self.client.get(url_for("datasets.recording_info", mbid=self.test_uuid))
         self.assert404(resp)
+
+    def test_evaluate_location_options(self):
+        self.temporary_login(self.test_user_id)
+        dataset_id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        resp = self.client.get(url_for("datasets.evaluate", dataset_id=dataset_id))
+        self.assertStatus(resp, 200)
```

Comment:
You should put a newline here to separate the fact that these are two separate tests